File: Validate-Subsequence/validatesub.java
import java.util.*;

class Program {
  public static boolean isValidSubsequence(List<Integer> array, List<Integer> sequence) {
    // Write your code here.
            int seqIndx = 0;
        int arrIndx = 0;
        while (seqIndx < sequence.size() && arrIndx < array.size()) {
            if (array.get(arrIndx) == sequence.get(seqIndx)) {
                seqIndx++;
            }
            arrIndx++;
        }
        if (sequence.size() == seqIndx) {
          return true;
        }
    return false;
  }
}

File: Validate-Subsequence/validatesub.py
def isValidSubsequence(array, sequence):

    
    # Write your code here.
    arrIndx = 0
    seqIndx = 0
    while arrIndx < len(array) and seqIndx < len(sequence):
        if array[arrIndx] == sequence[seqIndx]:
            seqIndx+=1
        arrIndx+=1
    return len(sequence) == seqIndx
    pass
